lib/modules/dosomething/dosomething_signup/dosomething_signup.module

```diff
@@ -229,11 +229,13 @@ function dosomething_signup_get_query_source() {
  * @param int $timestamp
  *   (optional) The timestamp of the signup.
  *   If not provided, uses @dries time.
+ * @param bool $transactional
+ *   (optional) Flag to trigger sending transactional messages related to signup.
  *
  * @return mixed
  *   The sid of the newly inserted signup, or FALSE if error.
  */
-function dosomething_signup_create($nid, $uid = NULL, $source = NULL, $timestamp = NULL) {
+function dosomething_signup_create($nid, $uid = NULL, $source = NULL, $timestamp = NULL, $transactional = TRUE) {
```

Comment:
I don't think we want the default here to be true, that means each time this is called throughout the codebase, it would be logged as a transactional signup